main.go

```diff
@@ -22,8 +21,6 @@ import (
 	"github.com/swaggo/gin-swagger/swaggerFiles"
 )
 
-var logBroadcaster broadcast.Broadcaster
```

Comment:
The `broadcast.Broadcaster` type from `github.com/dustin/go-broadcast` is still used inside `build.go`, it was just this `logBroadcaster` alias type that wasen't.